src/main/context-handler.ts

```diff
@@ -70,7 +70,8 @@ export class ContextHandler {
 
   async resolveAuthProxyUrl() {
     const proxyPort = await this.ensurePort();
-    return `http://127.0.0.1:${proxyPort}`;
+    const path = this.clusterUrl.path !== "/" ? this.clusterUrl.path : ""
```

Comment:
Why we have to avoid `/` in the end?
In other words, maybe always add `this.clusterUrl.path` to the end?

Reply:
`@kubernetes/client-node` does not like it

Reply:
Ok, thanks for info!